crates/esthri/Cargo.toml

```diff
@@ -40,12 +40,15 @@ tokio-retry = "0.3"
 tokio-stream = "0.1"
 tokio-util = { version = "0.7", features = ["io"] }
 walkdir = "2"
+strum = "0.24.1"
+strum_macros = "0.24.3"
 
 [dependencies.esthri-internals]
 path = "../esthri-internals"
 
 [dev-dependencies]
 esthri-test = { path = "../esthri-test" }
+esthri = { path = "../esthri"}
```

Comment:
do we need this

Reply:
nvm, this line breaks
`cargo make --profile dev+nativetls`
recursive dependency or something

Reply:
After encountering failed tests and fixing them, when only running the esthri integration test using "cargo make test -p esthri --test integration", cargo complains about the "esthri::blocking" mod not in scope. Adding this seems to resolve this issue but I don't think it's strictly needed, only as a convenient way to trigger small set of test

Reply:
try `cargo make test test_upload_storage_class_all`

Reply:
in `Makefile.toml`
```toml
[tasks.test]
env = { RUST_LOG = "esthri=debug", RUST_BACKTRACE = "1" }
command = "cargo"
args = [
        "test",
        "${@}",
        "--features=${TLS_PROVIDER}",
        "--no-default-features",
        "@@remove-empty(BUILD_FLAVOR)",
        "--",
        "@@remove-empty(TEST_FILTER)",
        "--nocapture"
        ]
```

the `${@}` should handle the paths so dont think you need to specify package `-p`

or just `cargo make test integration` should work exactly as  `cargo make test -p esthri --test integration` i believe

Reply:
dev dependency updated as suggested, ci passing